### DISCUSSION TITLE
Feature Request: Add information to the plugin store that denotes important feature support for each plugin

### DISCUSSION THREAD

**Discussion by jsunsawyer** [craftcms/console#74] [Plugin Store]
In the process of updating a huge site with several plugins over to 3.1. Not being able to tell if a plugin supports project config or environmental variables makes it sort of a guessing game.

It would be super handy if Craft would display a supported/unsupported list of any important features that are related to Craft.

This would also encourage plugin developers to support these optional Craft features, helping to create a consistent plugin ecosystem.

**Comment by brandonkelly** [comment]
This is sort of a duplicate of where we ended up in #3610, but I’ll leave this one open in favor of that since it’s more clear and concise.

**Comment by outline4** [comment]
I'm all for that!  

And a developer should also be able to point out which php extensions are needed on top of craft defaults.  
e.g. Solspace uses php extensions bcmath and amp.

**Comment by sidm1983** [comment]
I would just like to throw in my vote in support of this feature request. I was just thinking about this the other day and all of the above information about a plugin would be extremely handy.

Another bit of information would be how much of the craft GraphQL spec does the plugin support. Not sure if this would be useful as the handful of plugins I have tried all seem to work well with GraphQL, but just thought I'd add it to this feature request.

Thank you 🙌🏽🙏🏽
